src/components/breeze/sidebar_links.cr

```diff
@@ -0,0 +1,26 @@
+class Breeze::SidebarLinks < BaseComponent
+  needs context : HTTP::Server::Context
+
+  def render
+    link "Requests", to: Breeze::Requests::Index, class: link_class(Breeze::Requests::Index)
+    link "Queries", to: Breeze::Queries::Index, class: link_class(Breeze::Queries::Index)
+  end
```

Comment:
Also I could see this being customized with Habitat and giving it a component. This is also how I'd allow customizing pretty much *any* page in Breeze 🔥 

```crystal
Breeze::SidebarLinks.configure do |settings|
  settings.links << MyCustomLinkComponent.new
end

Breeze::RequestPage.configure do |settings|
  settings.sections << ->(req: BreezeRequest) { text: BreezePanel.new("#{req.id}") }
end
```

So the idea is people can use blocks and some of the breeze components to mix and match